packages/node/README.md

```diff
@@ -5,22 +5,25 @@ https://segment.com/docs/connections/sources/catalog/libraries/server/node/
 
 NOTE:  @segment/analytics-node is unstable! do not use.
 
-
+## Basic Usage
 ```ts
-// TODO: finalize API
-import { load } from '@segment/analytics-node'
+// analytics.ts
+import { AnalyticsNode } from '@segment/analytics-node'
 
+export const analytics = new AnalyticsNode({ writeKey: '<MY_WRITE_KEY>' })
 
-// some file
-export const ajsPromise = load({ writeKey: 'abc123' }).then(([ajs]) => ajs)
 
-// some other file
-import { ajsPromise } from '../analytics'
+// app.ts
+import { analytics } from './analytics'
 
-router.post('/user',  async (req, res) => {
-  ajsPromise.then((ajs) => ajs.track(user.id, "some event"))
-  res.json(user)
-})
+analytics.identify('Test User', { loggedIn: true }, { userId: "123456" })
+analytics.track('hello world', {}, { userId: "123456" })
 
+```
 
+# Event Emitter (Advanced Usage)
```

Comment:
More of a todo, but we should make sure to document how to handle errors. (maybe that won't be apparent to us until the plugin is added)

Reply:
I created a new `dispatchWithErrorHandling` function that is worth a look -- it emits an 'error' whenever there's a delivery failure due to for example, a validation failure, or an outright rejection (such as a network error)